third_party/terraform/website/docs/d/iam_workload_identity_pool.markdown

```diff
@@ -0,0 +1,35 @@
+---
+subcategory: "Cloud IAM"
+layout: "google"
+page_title: "Google: google_iam_workload_identity_pool"
+sidebar_current: "docs-google-datasource-iam-workload-identity-pool"
+description: |-
+  Get a IAM workload identity pool from Google Cloud
+---
+
+# google\_iam\_workload_\identity\_pool
+
+Get a IAM workload identity pool from Google Cloud by its id.
+
```

Comment:
We probably want a note that this resource is in beta only here:
```
~> **Warning:** This resource is in beta, and should be used with the terraform-provider-google-beta provider.
See [Provider Versions](https://terraform.io/docs/providers/google/guides/provider_versions.html) for more details on beta resources.
```
is the one that we generally use

Similar to: https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/compute_managed_ssl_certificate

Reply:
Done